src/components/FormSelectionContainer/FormSelect/FormSelect.tsx

```diff
@@ -13,55 +15,110 @@ interface FormSelectProps {
 
 export const FormSelect: FunctionComponent<FormSelectProps> = ({ id, form, onAddForm, ...props }) => {
   const { config } = useConfigContext();
-  const [errorMsg, setErrorMsg] = useState<string | null>(null);
-  const [isValidDns, setIsValidDns] = useState<boolean>();
+  const [tooltipMsg, setTooltipMsg] = useState<string | null>(null);
```

Comment:
think can further streamlined, remove this state and let text be determined by `formStatus` wdyt

Reply:
sorry i forget about multiple error messages my bad, see if this helps -> https://github.com/TradeTrust/document-creator-website/pull/230